tests/common/plugins/sanity_check/__init__.py

```diff
@@ -94,10 +94,12 @@ def do_checks(request, check_items, *args, **kwargs):
     for item in check_items:
         check_fixture = request.getfixturevalue(item)
         results = check_fixture(*args, **kwargs)
+        logger.debug("check results of each item {}".format(results))
```

Comment:
This is unnecessary.

Reply:
I think it's necessary. This log shows the form of the variable `results` before it is added into check_results.

Reply:
I got your point. Agree.